src/Display/ConvexHull.hs

```diff
@@ -1,16 +1,14 @@
+{-# LANGUAGE NamedFieldPuns #-}
+
 module Display.ConvexHull where
 
-import qualified Graphics.Rendering.Chart.Easy as C
-import qualified Graphics.Rendering.Chart.Backend.Cairo as C
+import qualified Graphics.Gloss as Gloss
+
+import ConvexHull.Common (Point (..), SimplePolygon (..))
+import Display.Display
 
-import ConvexHull.Common (Points (), ConvexHull ())
+instance Display Point where
+  render Point { x, y } = Gloss.translate (100 * x) (100 * y) $ Gloss.circleSolid 1.0
 
-plotHull :: String -> Points -> ConvexHull -> IO ()
```

Comment:
Worth moving:
```haskell
   Gloss.display
      (Gloss.InWindow "Geometric Algorithms" (512, 512) (10, 10))
      Gloss.white
      (Gloss.pictures
        [ Gloss.color (Gloss.greyN 0.7) . render $ hull1
        , render points
        ])
```

from `src/Main.hs` to back here? Think it's a little cleaner and makes it clear what's a core part of the experiments.